Dockerfiles/toolbox.Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM registry.fedoraproject.org/fedora-toolbox:38
 
-ARG GOLANG_VERSION=1.21
+ARG GOLANG_VERSION=1.22.9
```

Comment:
This was needed because it seems like that go.dev download site doesn't actually seem to maintain download links for x.y versions currently (maybe it did in the past?), so it seems like this was currently broken for 1.21.

Reply:
@ykaliuta in this particular Dockerfile, it's not for the go-toolset image: it uses this ARG as the version to download from the upstream go.dev site directly. See line 9

Reply:
> in this particular Dockerfile

Oops :)